bindings/python/test/test_mongocrypt.py

```diff
@@ -517,9 +512,8 @@ def test_encrypt_indexed(self):
         key_path = 'keys/ABCDEFAB123498761234123456789012-local-document.json'
         index_key_path = 'keys/12345678123498761234123456789012-local-document.json'
```

Comment:
Is `index_key_path` not needed at all anymore, here and in `test_rewrap_many_data_key`?

Reply:
I believe you are correct. Removed.